db/migrate/20240619135756_add_govuk_one_login_to_candidate_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddGovukOneLoginToCandidateTable < ActiveRecord::Migration[7.1]
+  def change
+    add_column :candidates, :govuk_one_login_uid, :string
```

Comment:
We have a mixture of `onelogin` and `one_login` we should be mindful in the production implementation to be consistent with our naming.